### PR TITLE
add selectors for text styles

### LINKED ISSUE #764: Feature Request: Color classes
We have color CSS variables, such as primary, secondary, etc.

I want to make text secondary like:
```
.text--secondary {
  color: var(--va-secondary);
}
```

There should be defined classes such as text-primary, bg-primary, etc.

### PR DESCRIPTION
**Pull request by sergey-kravcov**
close #764 

## Description
Added selectors for text styles next colors: "primary", "secondary", "success", "info", "danger", "warning", "gray", "dark".
Rule: --va-[color]
Снимок экрана 2021-07-07 в 17 24 44

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/ui/src/components/vuestic-sass/global/typography.demo.vue [hunk 1]
```diff
@@ -1,34 +1,42 @@
...
-    <VbCard>
+      <br>
       <a
         href="#"
         class="link-secondary"
       >Secondary link Visited</a>
     </VbCard>
+    <VbCard title="Text styles">
+      <div>Default Text</div>
+      <div v-for="color in textStyleColors" :key="color" :class="`text--${color}`">{{ color }} text</div>
```
Comment (m0ksem) [inline packages/ui/src/components/vuestic-sass/global/typography.demo.vue:26]: According to https://github.com/epicmaxco/vuestic-ui/issues/876, we should use the `va` prefix in every of our classes. Please, make sure that the feature will not conflict with other frameworks.

**Review by m0ksem — CHANGES_REQUESTED** [review]
Please, prevent conflicts with other frameworks.

Also, there should be few lines in docs about this feature.

**Comment by m0ksem** [comment]
If I am a user I can define my custom color `--va-strange` via `GlobalConfig > ColorConfig`. CSS variable will be created, but CSS class - not.

**Comment by sergey-kravcov** [comment]
> If I am a user I can define my custom color `--va-strange` via `GlobalConfig > ColorConfig`. CSS variable will be created, but CSS class - not.

After discussion it was decided to implement in the future

**Pull request closed without merging**